internal/message.go

```diff
@@ -10,16 +11,21 @@ import (
 	segmentio "github.com/segmentio/kafka-go"
 )
 
+const (
+	RetryHeaderKey       = "x-retry-count"
+	MessageTimeHeaderKey = "x-time"
```

Comment:
I don't sure what the correct name is. Any suggestion? @emreodabas

Reply:
may be x-produceTime